src/main/scala/li/cil/oc/server/component/DebugCard.scala

```diff
@@ -388,6 +388,16 @@ object DebugCard {
       null
     }
 
+    @Callback(doc = """function(x:number, y:number, z:number, sound:string, range:number) -- Play a sound at the specified coordinates.""")
+    def playSoundAt(context: Context, args: Arguments): Array[AnyRef] = {
+      checkEnabled()
+      val (x, y, z) = (args.checkInteger(0), args.checkInteger(1), args.checkInteger(2))
+      val sound = args.checkString(3)
+      val range = args.checkInteger(4)
+      world.playSoundEffect(x, y, z, sound, range / 15 + 0.5F, 1.0F)
```

Comment:
This is pretty much the playSoundAt from OpenSecurity, from my testing converting it to a float makes no difference, I don't remember why I didn't in the end though.